openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -583,14 +583,14 @@ const DatabaseSchemaPage: FunctionComponent = () => {
         }),
         dataIndex: 'name',
         key: 'name',
-        render: (text: string, record: Table) => {
+        render: (_, record: Table) => {
```

Comment:
Can we update dataIndex to displayName and it will work out of the box

Reply:
but We won't get a fallback if there is no displayName available, with the use of `{getEntityName(record)}` it will pick displayname and if its not available it will pick name